library/Freckle/App/Database.hs

```diff
@@ -57,6 +62,9 @@ class HasSqlPool app where
 instance HasSqlPool SqlPool where
   getSqlPool = id
 
+class HasActiveConnectionGauge app where
```

Comment:
I think you forgot to add this to exports. Gonna be hard to end-users to make the required instance. Personally, I'd also (re)export `Guage` and whatever constructor functions are required to make one. I'm not sure if the qualified import makes that annoying though.